Kentico.Kontent.Management.Tests/Kentico.Kontent.Management.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net5.0;net6.0</TargetFrameworks>
+    <TargetFrameworks>net6.0;net5.0</TargetFrameworks>
```

Comment:
Why did you switch targets?

Reply:
I was just testing something, but it did not work, and forgot to switch them back. Will do.